docs/DataFormats.md

```diff
@@ -0,0 +1,80 @@
+# Data Formats
+
+DIVE Desktop and Web support a number of annotation and configuration formats.  The following formats can be uploaded or imported alongside your media and will be automatically parsed.
+
+* DIVE Annotation JSON (default annotation format)
+* DIVE Configuration JSON
+* VIAME CSV
+* COCO and KWCOCO
+* KPF (Kitware Packet Format) for MEVA
+
+## DIVE Annotation JSON
+
+Files are typically named `result_{dataset-name}.json`.  This JSON file is a map of numeric track identifiers to tracks, or `Record<string, TrackData>`, where TrackData is defined below.
+
+``` typescript
+interface TrackData {
+  trackId: TrackId;
+  meta: StringKeyObject;
+  attributes: StringKeyObject;
+  confidencePairs: Array<ConfidencePair>;
+  features: Array<Feature>;
+  begin: number;
+  end: number;
+}
```

Comment:
Is this really helpful without defining what `Feature` and `ConfidencePair` are?  I know you can go to the link below and the confidencePair/Feature definitions are right there.  Confidence Pair I think can be simplified inline if required.  Feature though is probably the important thing people care about if they make it to this page.

Reply:
I agree. Let's reproduce the other interfaces here.